docs_build/tutorials_templates/data_management/cloud_storage/binding_with_azure/mds.py

```diff
@@ -14,41 +14,72 @@ def section1():
     ### Create the Blob Function
     1. Create a Container in the created Storage account
        * Public access level -> Container OR Blob
-    NOTE this container should be used as the external storage for the Dataloop dataset.
-    2. Go back to Resource group and click Create -> Function App
-       * Choose Subscription, your Resource group, Name and Region
+    NOTE: This container should be used as the external storage for the Dataloop dataset.
+    2. Go back to Function App and click create -> to create a new function
+       * Choose Subscription
+       * Choose your Resource group
+       * Choose Function name
        * Publish -> Code
        * Runtime stack -> Python
-       * Version -> <=3.7
-       * press next and choose your Storage account
+       * Version -> 3.10 >= Version >= 3.7
+       NOTE: when choosing python 3.7 please pay attention to the AOL warning
+       * Choose Region
+       * Use default values for all other options (OS and Plan ...)
+       * Press next and choose your Storage account
+       * Review and create
 
-    In VScode, flow the instructions in [azure docs](https://learn.microsoft.com/en-us/azure/azure-functions/create-first-function-vs-code-python) to configure your environment and deploy the function:
+    ### Deploy your function
+    In VS code, flow the instructions in [azure docs](https://learn.microsoft.com/en-us/azure/azure-functions/create-first-function-vs-code-python) to configure your environment and deploy the function:
     1. Configure your environment
     2. Sign in to Azure
     3. Create your local project
```

Comment:
* On VS code go to Azure panel -> workspace (bottom left panel)-> create a function